pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/SortOperator.java

```diff
@@ -134,8 +150,10 @@ private void consumeInputBlocks() {
         for (Object[] row : container) {
           SelectionOperatorUtils.addToPriorityQueue(row, _rows, _numRowsToKeep);
         }
-
+        _operatorStats.endTimer();
         block = _upstreamOperator.nextBlock();
+        _operatorStats.startTimer();
+        _operatorStats.recordInput(1, block.getNumRows());
```

Comment:
I noticed that in your output example `SORT` always reports zero input rows. Any idea why that might be?

Reply:
Maybe block.getNumRows() is wrong. I updated it to container.size() and verified it is working.